Dockerfile

```
@@ -0,0 +1,42 @@
+COPY . /app
+
+# Expose all necessary ports
+
+# Install MySQL and configure it
+RUN apt-get update && \
+    apt-get install -y openjdk-21-jdk mysql-server && \
+    mkdir -p /var/lib/mysql && \
+    chown -R mysql:mysql /var/lib/mysql
+
+# Configure MySQL root user with no password and create databases
+RUN service mysql start && \
```

Hadolint (2.12.0)

[info] For some bash commands it makes no sense running them in a Docker container like `ssh`, `vim`, `shutdown`, `service`, `ps`, `free`, `top`, `kill`, `mount`, `ifconfig`

(DL3001)